backend/core/src/auth/services/user.service.ts

```diff
@@ -123,6 +124,7 @@ export class UserService {
     qb.andWhere("user.id IN (:...distinctIDs)", {
       distinctIDs: distinctIDResult.items.map((elem) => elem.id),
     })
+    qb.orderBy("user.id")
```

Comment:
Can you help me understand why this happens twice?

Reply:
ahh good catch, it doesn't have to happen twice, it just needs to happen in the query that gets us the unique ids. updated to reflect that